src/structures/idt.rs

```diff
@@ -634,6 +634,18 @@ pub type PageFaultHandlerFunc =
 #[derive(Copy, Clone, Debug)]
 pub struct PageFaultHandlerFunc(());
 
+/// A segment not present handler function that pushes a selector error code.
+///
+/// This type alias is only usable with the `abi_x86_interrupt` feature enabled.
+#[cfg(feature = "abi_x86_interrupt")]
+pub type SegmentNotPresentHandlerFunc =
+    extern "x86-interrupt" fn(InterruptStackFrame, error_code: SelectorErrorCode);
+
+/// This type is not usable without the `abi_x86_interrupt` feature.
+#[cfg(not(feature = "abi_x86_interrupt"))]
+#[derive(Copy, Clone, Debug)]
+pub struct SegmentNotPresentHandlerFunc(());
+
```

Comment:
Remove (for now). Also when we add this back, note that the `#NP` exception isn't the only exception that uses this error code. 

`#TS`, `#NP`: `SelectorErrorCode`
`#SS`, `#GP`: `SelectorErrorCode` or zero (depending on context)
`#AC`, `#DF`: Error code always 0

`#PF`, `#CP`, `#VC`, `#SX`: Custom error code